shell/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "shell",
+  "version": "0.0.0",
+  "private": true,
+  "config": {
+    "port": "8080",
+    "host": "localhost"
+  },
+  "scripts": {
+    "start": "cross-env http-server -a ${npm_package_config_host} -p ${npm_package_config_port}",
```

Comment:
Add comment explaining that we're sort of using npm as a build system here, and we may want to rework this logic to fit within either sigh or gulp.

https://stackoverflow.com/questions/14221579/how-do-i-add-comments-to-package-json-for-npm-install